content/ja/docs/contributing/sig-practices.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 承認者およびメンテナーのための SIG のプラクティス
+linkTitle: SIG のプラクティス
+description: 承認者およびメンテナーがどのようにイシューやコントリビューションを管理するかを学びます。
+weight: 999
+default_lang_commit: 722beb5b9ecf8a603b2f5ae6128e3c3509f52fcf
+cSpell:ignore: chalin docsy
+---
+
+このページでは、承認者およびメンテナーが使用するガイドラインと一般的なプラクティスについて説明します。
+
+## オンボーディング {#onboarding}
+
+コントリビュータがドキュメントに対する責任の大きい役割（承認者、メンテナー）を引き受ける場合、既存の承認者およびメンテナーによってオンボーディングが行われます。
+
+- `docs-approvers`（または `docs-maintainers`）グループに追加されます
+- `#otel-comms`、`#otel-maintainers`、およびチーム内のプライベート Slack チャンネルに追加されます
+- [SIG Comm ミーティング](https://groups.google.com/a/opentelemetry.io/g/calendar-comms)と[メンテナーミーティング](https://groups.google.com/a/opentelemetry.io/g/calendar-maintainer-meeting)のカレンダー招待への登録を求められます
+- SIG Comm の現在のミーティング時間が適しているかを確認し、適さない場合は既存の承認者およびメンテナーと調整して、全員に適した時間を決定します
+- コントリビュータ向けのさまざまなリソースを確認します。
+  - [コミュニティリソース](https://github.com/open-telemetry/community/)。
+    特に、[コミュニティメンバーシップ](https://github.com/open-telemetry/community/blob/main/community-membership.md) と
+    [ソーシャルメディアガイド](https://github.com/open-telemetry/community/blob/main/social-media-guide.md) に関するドキュメント
+  - [コントリビューションガイドライン](/docs/contributing)。この一環として、これらのドキュメントをレビューし、イシューやプルリクエストを通じて改善するためのフィードバックを提供します
+
+以下は、追加で確認すると有益なリソースです。
+
+- [Hugo ドキュメント](https://gohugo.io/documentation/)
+- [Docsy ドキュメント](https://www.docsy.dev/docs/)
+- [マーケティングガイドライン](/community/marketing-guidelines/)。これは、Linux Foundation のブランドおよび
+  [商標使用ガイドライン](https://www.linuxfoundation.org/legal/trademark-usage) 含みます。
+  これらは、レジストリ、インテグレーション、ベンダー、導入事例、ディストリビューションのエントリーをレビューする際に特に重要です。
+
+## コラボレーション {#collaboration}
+
+- 承認者およびメンテナーは、それぞれ異なる勤務時間や状況です。
+  そのため、すべてのコミュニケーションは非同期とみなされ、通常のスケジュール外で返信する義務はありません
+- 承認者またはメンテナーが長期間（数日または1週間以上）不在となる場合は、
+  [#otel-comms](https://cloud-native.slack.com/archives/C02UN96HZH6) チャンネルでの通知や GitHub ステータスの更新を行います。
+- 承認者およびメンテナーは、[OTel 行動規範](https://github.com/open-telemetry/community/?tab=coc-ov-file#opentelemetry-community-code-of-conduct) と[コミュニティの価値観](/community/mission/#community-values)に従います。
+  彼らはコントリビュータに対して親切かつ協力的であり、何らかの対立や誤解、不快になるような状況が承認者またはメンテナーに生じた場合は、会話、イシューまたは PR から一歩引いて別の承認者やメンテナーに対応を依頼できます。
```

Comment:
```suggestion
  彼らはコントリビューターに対して親切かつ協力的であり、何らかの対立や誤解、不快になるような状況が承認者またはメンテナーに生じた場合は、会話、イシューまたは PR から一歩引いて別の承認者やメンテナーに対応を依頼できます。
```

Reply:
@ymotongpoo

Thank you for your review.
I fixed it in a below commit.

https://github.com/open-telemetry/opentelemetry.io/pull/6538/commits/9d3a844ba4c2b78a13608b6b893fa64f141e9822